src/js/page/ui/copy-button.js

```diff
@@ -35,9 +41,12 @@ export default class CopyButton extends FloatingActionButton {
     document.execCommand('copy');
     getSelection().removeAllRanges();
     document.body.removeChild(this._pre);
+    
+    return true;
   }
-
   setCopyText(text) {
     this._text = text;
   }
+  
```

Comment:
Nitpicking these 2 newlines are not needed :)

Reply:
This is why people invented automatic linting ;)